source/includes/_energy-ratings.md

```diff
@@ -30,10 +31,95 @@ curl https://api.gresb.com/api/v1/energy_ratings
 
 ### Create, Update and Remove Energy Rating
 
-To add an energy rating to an asset, send a request to `/entities/{entity_id}/assets/{asset_id}` using PATCH method. Energy ratings are posted annually as outlined in the data dictionary. Each energy rating has to have an ID and a size specified in order to be valid. Energy ratings can be updated with the corresponding `energy_rating_id` and removed by setting the related fields to `null`.
+Previously, participants could submit only one energy rating per asset. As per the 2024 Standard Changes, asset data entry form will be accomodating the submission of **multiple energy ratings per asset**, facilitating more accurate reporting especially meant for residential units.
+
+Unlike the previous structure where energy ratings were reported under the objects "energy_rating_id" and "energy_rating_size" in the annual_data array, they will now be nencapsulated within their own array, outside the `annual_data`, similar to Building Certifications.
+
+To add an energy rating to an asset, send a request to `/entities/{entity_id}/assets/{asset_id}` using PATCH method. Energy ratings are no longer posted annually as outlined in the data dictionary. Each energy rating has to have an ID and a size specified in order to be valid.
```

Comment:
This was wrongly written to begin with and should follow the same structure as certifications.

This means that you make a normal POST or PATCH request (which we don't have to specify) with the required request json structure, now including `energy_ratings`.

https://github.com/GRESB/api-docs/blob/master/source/includes/_certifications.md#create-certifications

We might even want to combine building certifications and energy ratings at this stage - given they are almost identical (besides the `level` for certifications.

Reply:
I will create ticket to combine the two sections, I removed mention of PATCH method and made the text more generic for now.